src/main/java/frc/robot/RobotContainer.java

```diff
@@ -130,6 +130,21 @@ private void configureButtonBindings() {
         // JoystickButton farm11 = new JoystickButton(m_farm, 11);
         // farm11.whenPressed(new ResetElevatorEncoder(m_climber));
 
+        // Additional manual shooter position buttons(orange buttons)
+        JoystickButton farm4 = new JoystickButton(m_farm, 4);
+        farm4.whenPressed(new ShooterCommand(m_shooter, m_intake, Constants.TARMAC_DEFAULT_DISTANCE, true));
```

Comment:
What is the use for this? It is the same shot as XBOX_A. Maybe have a shot further inside the Tarmac?

Reply:
Cognitive consistency. 

I'd be fine if we made this closer, but didn't have shoot numbers I trusted for that. Feel free to change in testing.